syntax = "proto3";

package auth;

service AuthService {
  rpc ValidateToken (TokenRequest) returns (UserResponse);
}

message TokenRequest {
  string token = 1;  // Keep the token as a simple string
}

message UserResponse {
  bool is_valid = 1;
  string user_id = 2;
  string email = 3;
}
